src/Hedgehog/Gen.fs

```diff
@@ -393,11 +393,16 @@ module Gen =
     let latin1 : Gen<char> =
         char '\000' '\255'
 
-    /// Generates a Unicode character, excluding invalid standalone surrogates:
-    /// '\000'..'\65535' (excluding '\55296'..'\57343')
+    /// Generates a Unicode character, excluding noncharacters('\65534', '\65535')
+    /// and invalid standalone surrogates ('\000'..'\65535' excluding '\55296'..'\57343')
```

Comment:
It'd be nice if you can keep the lines <= 79-80 chars width, which means this would then look like this:

```
/// Generates a Unicode character, excluding noncharacters
/// ('\65534', '\65535') and invalid standalone surrogates
/// ('\000'..'\65535' excluding '\55296'..'\57343').
```

Reply:
I did it.